app/dashboard/views.py

```diff
@@ -5836,3 +5837,36 @@ def showcase(request, hackathon):
     return JsonResponse({
         'success': True,
     })
+
+
+def events(request, hackathon):
+    hackathon_event = get_object_or_404(HackathonEvent, id=hackathon)
```

Comment:
as this seems to be a Rest endpoint with json answers lets keep the server answers to rest json payloads. In that way we can be sure the reason why failing is actually a not found event.

Reply:
done